paasta_tools/tron_tools.py

```diff
@@ -559,6 +559,19 @@ def get_node(self):
     def get_schedule(self):
         return self.config_dict.get("schedule")
 
+    def get_cron_expression(self) -> Optional[str]:
+        schedule = self.config_dict.get("schedule")
+        if (
+            isinstance(schedule, Dict)
+            and "type" in schedule
+            and schedule["type"] == "cron"
+        ):
+            return schedule["value"]
+        elif isinstance(schedule, str) and schedule.startswith("cron"):
+            return schedule
```

Comment:
suggestion: thoughts on returning
```suggestion
            return schedule.replace("cron", "")
```
here so that we don't need to do any additional manipulation of this value in `print_upcoming_runs()`?